### PR TITLE
Feature/restructure files controllers

### PR DESCRIPTION
**Pull request by danielronnkvist**
I have restructured how the files controllers are organized.
### before
- FilesController
  - BrowseController
  - DownloadController
### after
- FilesController
  - BrowseController
  - DownloadController
  - CreateController
    - CreateTextController
  - UploadController

### CODE REVIEW

File: app/controllers/files/browse_controller.rb [hunk 1]
```diff
@@ -1,90 +1,17 @@
...
   def browse
     path = params[:path] || '/'
 
     # setup client
-    @client = DropboxClient.new(@identity.token)
-    @client = Google::APIClient.new(
-      :application_name => 'Bruse',
-      :application_version => '1.0.0'
-      )
+    # @client = DropboxClient.new(@identity.token)
```
Comment (klaseskilson) [inline app/controllers/files/browse_controller.rb]: This could perhaps be removed?

Comment (danielronnkvist) [inline app/controllers/files/browse_controller.rb]: Thought the same thing :crystal_ball:

Comment (klaseskilson) [inline app/controllers/files/browse_controller.rb]: Cool :thought_balloon:

**Comment by klaseskilson** [comment]
Looks good to me! 🚌

`create_text`-controller is cute, but that makes sense since it has so many private functions.

**Comment by ronjagrosz** [comment]
:blue_car: :palm_tree: looks good!

**Pull request merged**